modules/get-started/pages/whats-new.adoc

```diff
@@ -6,6 +6,51 @@ This topic includes new content added from version 4.29.0 onwards.
 
 For a full list of product updates, and details of updates prior to version 4.29.0, see the https://github.com/redpanda-data/connect/releases[Redpanda Connect release notes^]. For new Cloud-specific features, see xref:redpanda-cloud:get-started:whats-new-cloud.adoc[].
 
+== Version 4.42.0
+
+This release introduces a xref:guides:bloblang/playground.adoc[Bloblang playground] where you can try out your Bloblang syntax and get immediate feedback. It also updates Redpanda Connect licensing checks for Self-Managed. Starting from version 4.42.0, you must xref:get-started:licensing.adoc[apply your Enterprise Edition license] to use any Enterprise connectors.
+
+== Version 4.41.0
+
+This release adds the `max_records_per_request` field to the xref:components:outputs/aws_sqs.adoc[`aws_sqs` output].
+
+=== Packaging update
+
+Starting from version 4.41.0, the `-cgo` suffixed Docker image is no longer available due to low demand and the unacceptable cadence with which the image base (Debian) receives security updates.
+
+To create your own CGO builds, run the following command:
+
+```bash
+CGO_ENABLED=1 make TAGS=x_benthos_extra redpanda-connect. 
+```
+
+== Version 4.40.0
+
+This update includes a beta release of the xref:components:inputs/pg_stream.adoc[`pg_stream` input] for Self-Managed and Cloud, which captures data changes made to a PostgreSQL database in real time and streams them to Redpanda Connect.
+
+=== Component updates
+
+This release adds:
+
+* The `metadata_max_age` field to the xref:components:outputs/redpanda_migrator_offsets.adoc[`redpanda_migrator_offsets` output].
```

Comment:
Maybe put this in a table to make it easy to scan.

Reply:
I'll open a ticket for this, so I can go back over all the other releases and make them consistent.